client/src/context/useProfilePhotoContext.tsx

```diff
@@ -0,0 +1,37 @@
+import { createContext, useState, FunctionComponent, useContext, useEffect } from 'react';
+import getProfilePhoto from '../helpers/APICalls/getProfilePhoto';
+
+interface IProfilePhotoContext {
```

Comment:
```suggestion
interface ProfilePhotoContext {
```
We don't need to prefix our interface with 'I' in JS